programs/server/HTTPHandlerFactory.cpp

```diff
@@ -28,7 +28,7 @@ HTTPRequestHandlerFactoryMain::HTTPRequestHandlerFactoryMain(const std::string &
 {
 }
 
-Poco::Net::HTTPRequestHandler * HTTPRequestHandlerFactoryMain::createRequestHandler(const Poco::Net::HTTPServerRequest & request) // override
+Poco::Net::HTTPRequestHandler * HTTPRequestHandlerFactoryMain::createRequestHandler(const Poco::Net::HTTPServerRequest & request)
```

Comment:
Why don't we return `unique_ptr` in all `createHandlerFactory` and `createRequestHandler` methods?

Reply:
It mimics Poco interface, but I'll try to check if we can improve it.

Reply:
It's override of a virtual method, we cannot improve here.